lib/error-debug.js

```diff
@@ -11,11 +11,20 @@ export default ({ err: { name, message, stack, module } }) => (
     {
       name === 'ModuleBuildError'
       ? <pre style={styles.message} dangerouslySetInnerHTML={{ __html: ansiHTML(encodeHtml(message)) }} />
-      : <pre style={styles.message}>{stack}</pre>
+      : <StackTrace error={err} />
     }
   </div>
 )
 
+const StackTrace = ({ error: { name, message, stack } }) => (
+  <div>
+    <div style={styles.heading}>{name && message && `${name}: ${message}`}</div>
```

Comment:
Sure:

<img width="782" alt="screen shot 2017-01-15 at 16 11 52" src="https://cloud.githubusercontent.com/assets/6324199/21963558/63daeb52-db3d-11e6-89ca-df8c1353de61.png">

We would always display it twice in chrome, maybe we should add the word `stacktrace:` above the stacktrace.